document_page_approval/models/document_page_history.py

```diff
@@ -156,14 +160,18 @@ def _compute_page_url(self):
             )
 
     def _compute_diff(self):
-        """Shows a diff between this version and the previous version"""
+        """Shows a cleaned-up diff between this version and the previous version."""
         history = self.env["document.page.history"]
         for rec in self:
             domain = [("page_id", "=", rec.page_id.id), ("state", "=", "approved")]
             if rec.approved_date:
                 domain.append(("approved_date", "<", rec.approved_date))
             prev = history.search(domain, limit=1, order="approved_date DESC")
             if prev:
-                rec.diff = self._get_diff(prev.id, rec.id)
+                raw_diff = self._get_diff(prev.id, rec.id)
             else:
-                rec.diff = self._get_diff(False, rec.id)
+                raw_diff = self._get_diff(False, rec.id)
+
+            cleaned_diff = re.sub(r'data-last-history-steps=".*?"', "", raw_diff)
+
+            rec.diff = cleaned_diff
```

Comment:
What is the reason for this? Sorry, but I don't understand (if it is necessary, perhaps it would be appropriate to explain in the method the reasons, so that we all know it in the future).